wallet-new/test/unit/Wallet/Abstract.hs

```diff
@@ -119,6 +119,22 @@ interpret es p = go
     verify :: Wallets ws h a -> Either Text (Wallets ws h a)
     verify ws = p ws >> return ws
 
+inductiveGen 
+    :: Gen (Block h a)
+    -> Gen (Transaction h a)
```

Comment:
Naturally, the "hard part" here is coming up with these generators :slightly_smiling_face: